internal/middleware/validator/validator.go

```diff
@@ -0,0 +1,54 @@
+package validator
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+)
+
+type ctxKey string
+
+var (
+	requestIsValidatedCtxKey = ctxKey("request-validated")
+)
+
+func ContextWithRequestIsValidated(ctx context.Context) context.Context {
+	return context.WithValue(ctx, requestIsValidatedCtxKey, true)
```

Comment:
what if we use `grpc_ctxtags.Extract(r.ctx).Set(requestIsValidatedCtxKey true)`? then we’d get the `request-validated` [ field in the logs for free](https://github.com/openfga/openfga/blob/84c25cf6015f8a604aa87982c7c79d8ea634ca00/pkg/middleware/logging/logging.go#L57) (and we can add a test for this [here](https://github.com/openfga/openfga/blob/1f9c6370976a56dc0535309f02e3a12c4880a372/tests/check/check_test.go#L201-L213))

Reply:
Never mind, this isn't necessary